### DISCUSSION TITLE
PreferGetMethod and encoding

### DISCUSSION THREAD

**Discussion by tylerlianghuang** [urql-graphql/urql#1848] [Q&A] [answered]
Hi, I'm using PreferGetMethod for a Get Query that includes a string with commas e.g. `"crop,progressive,format=webp"`.

However, during the encoding process, this [line](https://github.com/FormidableLabs/urql/blob/ef4b5b002d31e17d84fb41e5feb24d2200c9656f/packages/core/src/internal/fetchOptions.ts#L44) of code replaces commas with whitespace and then encoded it with `%20`. 

It makes our query become invalid and I am wondering whether it's worth to raise a issue to exclude comma in the regex. Since this [PR](https://github.com/FormidableLabs/urql/pull/908) only mentions whitespace.

**Comment by JoviDeCroock** [comment, marked as answer]
I wonder if it'd be better to pass these in as `variables` 😅  all though it's a good point, maybe it's worth transitioning to the `stripIgnoredCharacters` helper from `graphql` or removing `,` from there as it might generally lead to some unexpected scenario's.

EDIT: will be solved by https://github.com/FormidableLabs/urql/pull/1851
